packages/ckeditor5-core/src/editor/editorconfig.ts

```diff
@@ -31,17 +31,16 @@ import type { MenuBarConfig } from '@ckeditor/ckeditor5-ui';
  * 	.then( ... )
  * 	.catch( ... );
  * ```
- *
- * Check the {@glink getting-started/legacy/installation-methods/predefined-builds Configuration} guide for more information
- * about setting configuration options.
  */
 export interface EditorConfig {
 	context?: Context;
 
 	/**
+	 * ------TODO redirect to proper guide--------
```

Comment:
@witek, @godai78 I would like to ask your opinion how to handle this part.

Reply:
IMHO we should describe what this method does, and AFAIK it's available on all editor types. Something like:  
\``he list of additional plugins to load along those already available in the editor. This could be used when someone is using multiple editors on a page, and extends the of the shared editor class`